kubernetes/Makefile

```diff
@@ -79,6 +108,23 @@ build/vars-merged-with-secrets.yml: var.yml var-$(CLUSTER).yml $(CONCENT_SECRET_
 build/image-version.yml:
 	echo "image_version: $(IMAGE_VERSION)" >> build/image-version.yml
 
+build/control-database-configuration.yml:
+	echo "selected_database:      default"  >> "$@"
+	echo "database_name:         \"{{ control_database_name }}\"" >> "$@"
+	echo "database_user:         \"{{ control_database_user }}\"" >> "$@"
+	echo "database_host:         \"{{ control_database_host }}\"" >> "$@"
+	echo "django_admin_fixtures: true" >> "$@"
+	echo "db_admin_password:     \" {{ control_db_admin_password }}\"" >> "$@"
+	echo "db_user_password:      \" {{ control_db_user_password }}\"" >> "$@"
+
+build/storage-database-configuration.yml:
+	echo "selected_database:   storage" >> "$@"
```

Comment:
Please rename it to `selected_database_id`. It might make it clearer that it's not the database name.

Reply:
Done.